data/successors/successor news.txt

```diff
@@ -0,0 +1,277 @@
+		word
+			`"`
+
+
+news "high houses official"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		not attributes crime uninhabited
+	name
+		word
+			"High Houses official"
+			"High Houses officer"
+	message
+		word
+			`"`
+		word
+			"The Houses welcome thee as 'guest' among our people; "
+			"Please remain in concord with our people; "
+			"Informed, I am, of thy recent arrival; "
+			"Can I provide assistance, honored traveler? "
+			"Our noble house does greet thee full of joy; "
+		word
+			"I am at thy service if 'tis needed."
+			"Today's been rather busy, please speak quickly."
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"Please submit to scans if 'tis requested"
+			"Taxes on transactions are collected"
+			"Transactions in this area are surveilled"
+			"If thou see'st something suspect, report't promptly"
+		word
+			"; port rules and regulations should be posted."
+			" in accordance with our standard port procedure."
+		word
+			`"`
+
+
+news "successor student"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		attributes education wealthy
+		not attributes uninhabited
+	name
+		word
+			"Successor student"
+	message
+		word
+			`"`
+		word
+			"Exams, I think, shall never know their end,"
+			"Hath I forgot my geographic terms?"
+			"Hath I forgot my chemical reagents?"
+			"Hyperspace Mechanics is a bore,"
+			"Predecessors His'try is a drag,"
+			"Plasma Flow Dynamics is a bore,"
+			"Particle Kinetics is a bore,"
+		word
+			" "
+		word
+			"I know not sleep nor food nor meager rest"
+			"I wish that I could learn while I'm asleep"
+			"I don't know how I even stay awake"
+			"I wish I knew some better study tricks"
+			"but I know that I'm the best one in my class"
+		word
+			"."
+			"!"
+		word
+			`"`
+
+
+news "successor merchant"
+	to show
+		has "language: Successor"
+	location
+		attributes successor
+		not attributes uninhabited
+	name
+		word
+			"Independent merchant"
+	message
+		word
+			`"`
+		word
+			"Interested in "
+		phrase
+			"successor commodities (2-syllable)"
+		word
+			", noble traveler?"
+			", canny buyer?"
+			", noble captain?"
+			", clever buyer?"
+			", newest friend?"
+			", thoughtful buyer?"
+			", thoughtful captain?"
+		word
+			`"`
+	message
+		word
+			`"`
+		word
+			"I miss the days when I had been a "
+			"I miss the times when I had been a "
+			"I still think back to when I was a "
+			"Thou remindeth me of when I was a "
+		word
+			"pilot, "
+			"captain, "
+			"ranger, "
+		word
+			"but alas "
+		word
+			"now's planetside I shall remain."
+			"'twas never brave enough to stay."
+			"I was a spacesick little knave."
+			"it's down at earth I feel at home."
+			"it's selling items that's for me."
+			"my family needed me return."
+			"possessed of talent I was not."
+		word
+			`"`
+	message
+		word
+			"A "
+		word
+			"pushy "
+			"unscrupulous "
```

Comment:
This one and `anxious-looking` don't work with the article `a`. I'd probably just remove them.